scripts/gulpfiles/build_tasks.js

```diff
@@ -409,30 +411,58 @@ goog.require('Blockly.requires');
 };
 
 /**
- * This task builds Blockly's lang files.
- *     msg/*.js
+ * This task regenrates msg/json/en.js and msg/json/qqq.js from
+ * msg/messages.js.
  */
-function buildLangfiles(done) {
+function generateLangfiles(done) {
   // Run js_to_json.py
-  const jsToJsonCmd = `python ./scripts/i18n/js_to_json.py \
---input_file ${path.join('msg', 'messages.js')} \
---output_dir ${path.join('msg', 'json')} \
---quiet`;
+  const jsToJsonCmd = `python scripts/i18n/js_to_json.py \
+      --input_file ${path.join('msg', 'messages.js')} \
+      --output_dir ${path.join('msg', 'json')} \
+      --quiet`;
```

Comment:
Minor note: I don't think these lines needed to be indented, if it was for the purpose of following the style guide.
Template literals specifically don't have to follow the indentation rules.
https://google.github.io/styleguide/jsguide.html#features-strings-template-strings

I don't think it hurts to change it, it'll just add extra spaces between the command line flags.

Reply:
Agreed.  I indented it here because that style rule continues with "though it may if the added whitespace does not matter", and we're not in any danger of exceeding the maximum command line length (8kiB on Windows and >200kB on macOS / Linux).